lrauv_ignition_plugins/proto/lrauv_state.proto

```diff
@@ -167,4 +167,16 @@ message LRAUVState
   // 0: Data collected from Chlorophyll sensor. Unit: ug / L
   // 1: Pressure calculated from current depth and latitude. Unit: Pa
   repeated float values_          = 32;
+
+  /// \brief Voltage of the battery.
+  double batteryVoltage_           = 33;
+
+  /// \brief Current supplied by the battery in A.
+  double batteryCurrent_           = 34;
+
+  /// \brief Current charge in the battery in Ah.
+  double batteryCharge_            = 35;
+
+  /// \brief Percentage of battery remaining.
+  double batteryPercentage_         = 36;
```

Comment:
@adityapande-1995 meta: hmm, I thought we were going to use the battery topic itself instead of tunneling this information through the state topic. @caguero ?

Reply:
We could do it that way, currently I just kept it consistent with how all other data streams are combined and supplied as one topic. Bridging it is easier on the lrauv side. Maybe @braanan / @caguero can comment on this ?

Reply:
I thought that was the plan as well. I'm ok with using the state message if that's simpler. The main downside I see here is backward compatibility, but that won't matter after we sync osrf/lrauv with lrauv-app

Reply:
Sorry for the confusion ! Should we merge this one and the one on the app ? Both of them have been approved.

Reply:
No worries! Yes, let's merge - I'll push the updated image short after. Thanks

Reply:
Merging ! :rocket: